boards/tenstorrent/tt_blackhole/tt_blackhole_fixed_partitions.dtsi

```diff
@@ -14,101 +20,171 @@ partitions {
 	flash-device-size = <DT_SIZE_M(64)>;
 	flash-block-size = <DT_SIZE_K(4)>;
 
-	cmfwcfg: partition@14000 {
+	/* Firmware configuration blobs */
+	cmfwcfg@14000 {
 		label = "cmfwcfg";
 		reg = <0x14000 DT_SIZE_K(4)>;
 		binary-path = "$BUILD_DIR/smc/generated_board_cfg/$BOARD_REV/fw_table.bin";
 		read-only;
 	};
 
-	origcfg: partition@15000 {
+	origcfg@15000 {
 		label = "origcfg";
 		reg = <0x15000 DT_SIZE_K(4)>;
 		binary-path = "$BUILD_DIR/smc/generated_board_cfg/$BOARD_REV/fw_table.bin";
 		read-only;
 	};
 
-	cmfw: partition@10000000 {
-		label = "cmfw";
-		reg = <0x10000000 DT_SIZE_K(106)>;
-		binary-path = "$BUILD_DIR/smc/zephyr/zephyr.bin";
-	};
-
-	ethfwcfg: partition@30000 {
+	ethfwcfg@16000 {
 		label = "ethfwcfg";
-		reg = <0x30000 DT_SIZE_K(4)>;
+		reg = <0x16000 DT_SIZE_K(4)>;
 		binary-path = "$BLOBS_DIR/tt_blackhole_erisc_params.bin";
 		read-only;
 	};
 
-	ethfw: partition@31000 {
-		label = "ethfw";
-		reg = <0x31000 DT_SIZE_K(64)>;
-		binary-path = "$BLOBS_DIR/tt_blackhole_erisc.bin";
+	memfwcfg@17000 {
+		label = "memfwcfg";
+		reg = <0x17000 DT_SIZE_K(4)>;
+		binary-path = "$BLOBS_DIR/tt_blackhole_gddr_params_$BOARD_REV.bin";
 		read-only;
 	};
 
-	memfwcfg: partition@41000 {
-		label = "memfwcfg";
-		reg = <0x41000 DT_SIZE_K(4)>;
-		binary-path = "$BLOBS_DIR/tt_blackhole_gddr_params_$BOARD_REV.bin";
+	ethsdreg@18000 {
+		label = "ethsdreg";
+		reg = <0x18000 DT_SIZE_K(4)>;
+		binary-path = "$BLOBS_DIR/tt_blackhole_serdes_eth_fwreg.bin";
 		read-only;
 	};
 
-	memfw: partition@42000 {
-		label = "memfw";
-		reg = <0x42000 DT_SIZE_K(16)>;
-		binary-path = "$BLOBS_DIR/tt_blackhole_gddr_init.bin";
+	flshinfo@19000 {
+		label = "flshinfo";
+		reg = <0x19000 DT_SIZE_K(4)>;
+		binary-path =
+			"$BUILD_DIR/smc/generated_board_cfg/$BOARD_REV/flash_info.bin";
 		read-only;
 	};
 
-	ethsdreg: partition@46000 {
-		label = "ethsdreg";
-		reg = <0x46000 DT_SIZE_K(4)>;
-		binary-path = "$BLOBS_DIR/tt_blackhole_serdes_eth_fwreg.bin";
+	/* Remote chip firmware images */
+	ethfw@1a000 {
+		label = "ethfw";
+		reg = <0x1a000 DT_SIZE_K(64)>;
+		binary-path = "$BLOBS_DIR/tt_blackhole_erisc.bin";
 		read-only;
 	};
 
-	ethsdfw: partition@47000 {
+	memfw@2a000 {
+		label = "memfw";
+		reg = <0x2a000 DT_SIZE_K(64)>;
+		binary-path = "$BLOBS_DIR/tt_blackhole_gddr_init.bin";
+		read-only;
+	};
+
+	ethsdfw@3a000 {
 		label = "ethsdfw";
-		reg = <0x47000 DT_SIZE_K(20)>;
+		reg = <0x3a000 DT_SIZE_K(64)>;
 		binary-path = "$BLOBS_DIR/tt_blackhole_serdes_eth_fw.bin";
 		read-only;
 	};
 
-	bmfw: partition@4c000 {
-		label = "bmfw";
-		reg = <0x4c000 DT_SIZE_K(68)>;
-		binary-path = "$BUILD_DIR/dmc/zephyr/zephyr.signed.bin";
-		read-only;
+	/* Update firmware image for the DMC */
+	dmfw: partition@4a000 {
+		label = "dmfw";
+		reg = <0x4a000 DT_SIZE_K(512)>;
```

Comment:
Not sure if MCUBoot would complain about this size, but I had it as 440 kiB here
https://github.com/tenstorrent/tt-zephyr-platforms/blob/de400d5fb85dd57c57f105700fe5d13a77d17214/boards/tenstorrent/tt_blackhole/tt_blackhole_fixed_partitions.dtsi#L108

Reply:
Not sure either. I'll make it one section (4KB) larger than the planned internal DMFW flash size (448 KB). So 452KB for this slot

Reply:
Done